src/apps/btc/btc_common.h

```diff
@@ -132,4 +132,9 @@ USE_RESULT bool btc_common_pkscript_from_outputhash(
     uint8_t* pk_script,
     size_t* pk_script_len);
 
+/**
+ * Converts between an xpub provided via a protobuf message to a libwally xpub.
+ */
+USE_RESULT bool btc_common_xpub_from_protobuf(const XPub* xpub_in, struct ext_key* xpub_out);
```

Comment:
Should this really live in the "btc" namespace? Doesn't seem only related to BTC.